### PR TITLE
Refactoring categories

### PR DESCRIPTION
**Pull request by rikirenz**
We are going to create 2 fields:

- One for the INSPIRE categories
- One for the arXiv categories

### CODE REVIEW

**Comment by rikirenz** [comment]
Ready for review! @david-caro

File: inspire_schemas/records/elements/arxiv_field.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (david-caro) [inline inspire_schemas/records/elements/arxiv_field.json]:
```
{
    'items': {
        'oneOf': [
            {
                'enum': ['one', 'two'],
                'type': 'string'
            }
        ]
    },
    'type': 'array',
    'uniqueItems': True
}

```

**Comment by rikirenz** [comment]
let's try again!

**Comment by david-caro** [comment]
This looks great now 👍 , but we should wait for #41 to see if this is actually what it's needed before merging, though we can start modifying the rest of the code (inspire-next + hepcrawl) to adapt to this in the meantime, as if not like this, it will be quite similar I think.

**Comment by david-caro** [comment]
I forgot! this is a non-backwards compatible change, so you should use this:
http://autosemver.readthedocs.io/en/latest/usage.html#major-change

**Comment by rikirenz** [comment]
@david-caro again new changes

**Comment by david-caro** [comment]
+ 💯 !

**Pull request merged**